lib/python/qmk/commands.py

```diff
@@ -101,6 +101,12 @@ def dump_lines(output_file, lines, quiet=True):
     if output_file and output_file.name != '-':
         output_file.parent.mkdir(parents=True, exist_ok=True)
         if output_file.exists():
+            with open(output_file, 'r', encoding='utf-8', newline='\n') as f:
```

Comment:
Would the `newline='\n'` part here not depend on <https://github.com/qmk/qmk_firmware/pull/23954> which went to develop?

Reply:
Perhaps; end result would be behaviour that matches the current state on `master` though -- we write the file every time.
Do you think it'd be worth reading then splitting, trimming and comparing both arrays for equality instead?

Reply:
> Do you think it'd be worth reading then splitting, trimming and comparing both arrays for equality instead?

Nah, only thing i would consider would be cherry-picking 23954 to master earlier than develop merge.